### DISCUSSION TITLE
last update no longer displaying

### DISCUSSION THREAD

**Discussion by cheeto12** [MMRIZE/MMM-CalendarExt3Journal#8] [General]
I updated MMM-CalendarExt3Journal and MMM-CalendarExt3Timeline; Timeline works fine but Journal disappeared. My config hasn't changed, and I'm not sure why it would just go away like that. Still poking around, but if anyone else has ideas I'm open to it.

**Comment by eouia** [comment]
Share me your config.js (including real ics url/file)

**Reply by cheeto12** [reply]
thanks! didn't have too much of a chance yesterday to keep working on it, but back at it again

**Reply by eouia** [reply]
<img width="643" alt="image" src="https://github.com/MMRIZE/MMM-CalendarExt3Journal/assets/1720610/71b9511b-4615-4a85-9304-545baef6ce7c">
This is what I got from your config. Is this not expected? What should it look like to-be?
(I changed `address` and `ipWhitelist` to my environment. And I'm not using burn-in module, so it makes error, but no matter.)

**Reply by cheeto12** [reply]
Yeah, that's what it should look like. For some reason not doing it on my end. Good to know its something I borked up and not the module in that case. 

Thanks for your help.

**Reply by cheeto12** [reply]
Sorry, I meant that *I* didn't have time to work on it yesterday, not you. My real job got in the way.

**Comment by eouia** [comment]
You can try to test modules one-by-one.
